src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -516,3 +531,74 @@ def generate_id_verification_code() -> str:
     Generate one-time identity verification code
     """
     return str(random.choice(range(100000, 999999)))
+
+
+def _retrieve_child_results(  # pylint: disable=R0911
+    fides_connector: Tuple[str, ConnectionConfig], rule_key: str
+) -> Optional[Dict[str, Optional[List[Row]]]]:
+    """Get child access request results to add to upload."""
+    secrets = fides_connector[1].secrets
+    if not secrets:
+        logger.error(
+            "No server secrets present for privacy request %s", fides_connector[0]
+        )
+        return None
+
+    uri = secrets.get("uri")
+    if not uri:
+        logger.error("No server uri present for privacy request %s", fides_connector[0])
+        return None
+
+    username = secrets.get("username")
+    if not username:
+        logger.error(
+            "No server user name present for privacy request %s", fides_connector[0]
+        )
+        return None
+
+    password = secrets.get("password")
+    if not password:
+        logger.error(
+            "No server password present for privacy request %s", fides_connector[0]
+        )
+        return None
+
+    client = FidesClient(
+        uri=uri,
+        username=username,
+        password=password,
+    )
+    try:
+        client.login()
+    except requests.exceptions.HTTPError as e:
+        logger.error(
+            "Error logging into to child server for privacy request %s: %s",
+            fides_connector[0],
+            e,
+        )
+        return None
+
+    try:
+        request = client.authenticated_request(
+            method="get",
+            path=f"{V1_URL_PREFIX}{PRIVACY_REQUEST_TRANSFER_TO_PARENT.format(privacy_request_id=fides_connector[0], rule_key=rule_key)}",
+            headers={"Authorization": f"Bearer {client.token}"},
+        )
+        response = client.session.send(request)
+    except requests.exceptions.HTTPError as e:
+        logger.error(
+            "Error retrieving data from child server for privacy request %s: %s",
+            fides_connector[0],
+            e,
+        )
+        return None
+
+    if response.status_code != 200:
+        logger.error(
+            "Error retrieving data from child server for privacy request %s: %s",
+            fides_connector[0],
+            response.json(),
+        )
+        return None
+
+    return {fides_connector[1].key: response.json()}
```

Comment:
i'm not sure this is the right key? at first glance i'm not 100% sure on the shape of the response data, so i may be mistaken - but i don't think we'll want to put the connector key as the key for this data. shouldn't the response data be "flattened" onto the aggregate results?

will step through it with real data on my end locally tomorrow to get a firmer understanding here..